rclcpp/include/rclcpp/node_interfaces/node_base.hpp

```diff
@@ -56,6 +56,11 @@ class NodeBase : public NodeBaseInterface
   const char *
   get_namespace() const;
 
+  RCLCPP_PUBLIC
+  virtual
```

Comment:
recommend `override` instead of `virtual` in subclass. Not really necessary though since it is an abstract function

Reply:
`virtual` is consistent with the rest of the methods in the file, which is why I choose to use it.